### DISCUSSION TITLE
Add import from OpenApi yaml

### DISCUSSION THREAD

**Discussion by marcosstefani** [Kong/insomnia#5773] [Ideas]
Much of the backend documentation using Java mainly with SpringBoot (but not only) uses the [openapi](https://swagger.io/specification/) documentation to develop an API documentation. Basically, a yaml file is generated where the entire API of a service is documented. It would be interesting to be able, when creating a new folder, to have the option of importing from a yaml file or a link that will give access to a yaml file that in turn can be imported by Insomnia. This would be a step up from any other tool. Attached is an example of documentation.
[api-docs.yaml.zip](https://github.com/Kong/insomnia/files/10717372/api-docs.yaml.zip)

**Comment by requizm** [comment]
I thought this already existed.

![Screenshot](https://github.com/Kong/insomnia/assets/12533045/d81b217b-8158-4067-a121-a6ee8701b61f)

**Comment by marcosstefani** [comment]
Thank you
